spec/lib/alpha_taxonomy/taxon_renamer_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe AlphaTaxonomy::TaxonRenamer do
+  describe "#run!" do
+    context "given some taxon base paths might require change" do
+      def test_payload(title, base_path)
+        {
+          title: title,
+          base_path: base_path,
+          publishing_app: 'collections-publisher',
+          rendering_app: 'collections',
+          format: 'taxon',
+          routes: [
+            { path: base_path, type: "exact" },
+          ]
+        }
+      end
+
+      it "requests Publishing API to change title and base_path" do
```

Comment:
It would be good to test the payload against the schema like:

https://github.com/alphagov/content-tagger/blob/4c52589e4ef8b24aefdda26eaa22df0db20dd23b/spec/lib/alpha_taxonomy/taxon_creator_spec.rb#L89-L91